lib/feeder_importer.rb

```diff
@@ -212,6 +212,11 @@ def template_for_episode(episode)
     episode.contains_video? ? video_template : audio_template
   end
 
+  def episode_description(episode)
+    attr = [:content, :summary, :description, :subtitle, :title].find { |d| !episode[d].blank? }
```

Comment:
is `attr` a reserved word in ruby and/or rails? [some](http://www.rubymagic.org/posts/ruby-and-rails-reserved-words) [sleuthing](http://reservedwords.herokuapp.com/) points to no, but we avoided using it in [the similar method](https://github.com/PRX/cms.prx.org/blob/master/app/models/episode_import.rb#L174) in EpisodeImport.rb, and Atom and Github are both giving it special syntax highlighting!